user-guide/Standard_Apps/Monitoring_Solutions/META_WhatsApp_Messaging/META_WhatsApp_Messaging_Installation.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: META_WhatsApp_Messaging_Installation
+---
+
+# Installing the META WhatsApp Messaging solution
+
+## Prerequisites
+
+- Make sure you are [registered as a Meta Developer](https://developers.facebook.com/docs/development/register).
```

Comment:
I attempted to test this, but I encountered an issue where I didn't receive the verification SMS at the start of the sign-up process, preventing me from becoming a Meta Developer. This might be an isolated problem, but I hope it's not something our users will face as well.

Reply:
I'll ask my teammates to follow the whole process to see if they have any problem too. I hope yours is an isolated case.
I'll get you back with the outcome.